src/sagemaker/deserializers.py

```diff
@@ -60,3 +62,34 @@ def deserialize(self, data, content_type):
             return data.read()
         finally:
             data.close()
+
+
+class CSVDeserializer(BaseDeserializer):
+    """Deserialize a stream of bytes into a list of lists."""
+
+    ACCEPT = "test/csv"
+
+    def __init__(self, encoding="UTF-8"):
```

Comment:
nit: probably most common [alias](https://docs.python.org/3/library/codecs.html#standard-encodings) used in the [docs](https://docs.python.org/3/library/stdtypes.html#bytearray.decode), in the wild, and even in the [cpython codecs implementation](https://github.com/python/cpython/blob/3.8/Lib/codecs.py#L1123) is lowercase `"utf-8"`

Reply:
Fixed